src/design-system/color/palettes.ts

```diff
@@ -2,12 +2,14 @@ import { mapValues } from 'lodash';
 
 export const colors = {
   appleBlue: '#0E76FD',
+  appleBlueLight: '#1F87FF',
   black: '#000000',
   blackTint: '#12131a',
-  grey: 'rgb(60, 66, 66)',
+  grey: '#3C4252',
```

Comment:
Just double checking — this is now a different colour from the original `rgb` value (its hex value is `#3C4242` rather than `#3C4252`), which means it's different from the opacity variants defined below. Is this deliberate? Do the opacity variants need to be updated too?

Reply:
Looks like this got updated from Christian's profiles polish PR. Looks like the variants on the design spec match `#3C4252`. I'll change the variants to match accordingly.

Reply:
changed back